services/movie_session.py

```diff
@@ -42,3 +42,13 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Similar to the update function, consider handling the case where the MovieSession with the given 'session_id' does not exist to avoid unhandled exceptions.